backend/app/Models/User.php

```diff
@@ -164,6 +165,26 @@ public function submissions(): BelongsToMany
             ->withPivot(['id', 'user_id', 'role_id', 'submission_id']);
     }
 
+    /**
+     * Inline comments that belong to the user
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function inlineComments(): HasMany
+    {
+        return $this->hasMany(InlineComment::class, 'created_by');
+    }
+
+    /**
+     * Overall comments that belong to the user
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function overallComments(): HasMany
+    {
+        return $this->hasMany(OverallComment::class, 'created_by');
+    }
+
```

Comment:
```suggestion
```
This wasn't part of the spec.   If I'm missing something thats been defined as a feature so far that needs this relationship established, I'll happily revisit.